source/org/zfin/marker/agr/GeneLinkMLInfo.java

```diff
@@ -0,0 +1,93 @@
+package org.zfin.marker.agr;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import org.alliancegenome.curation_api.model.ingest.dto.IngestDTO;
+import org.zfin.infrastructure.ActiveData;
+import org.zfin.marker.Marker;
+import org.zfin.ontology.datatransfer.AbstractScriptWrapper;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.zfin.repository.RepositoryFactory.getMarkerRepository;
+
+public class GeneLinkMLInfo extends AbstractScriptWrapper {
+
+    private int numfOfRecords = 0;
+
+    public GeneLinkMLInfo(int number) {
+        numfOfRecords = number;
+    }
+
+
+    public static void main(String[] args) throws IOException {
+        int number = 0;
+        if (args.length > 0) {
+            number = Integer.valueOf(args[0]);
```

Comment:
Where does args[0] get set? I remember gradle is weird about passing along command line arguments. Also, I think args[1] is the first argument and args[0] is the command being executed?